src/lib/y2storage/fake_device_factory.rb

```diff
@@ -657,6 +657,11 @@ def create_free(parent, args)
       disk_name
     end
 
+    ENCRYPTION_METHOD_ALIASES = {
+      "luks"  => "luks1",
+      "luks2" => "pervasive_luks2"
```

Comment:
Not sure whether this alias is a good idea. What will happen when we implement general support for LUKS2?